setup.py

```diff
@@ -15,8 +15,13 @@
           'requests==2.20.0',
           'backoff==1.3.2',
           'requests_mock==1.3.0',
-          'nose'
       ],
+      extras_require= {
+          'dev': [
+              'pylint==2.5.3',
```

Comment:
I pinned this because I saw failures in the standard list of disables on `pylint v2.6.0`